pkg/hook/hook.go

```diff
@@ -260,3 +271,14 @@ func (h *Hook) prepareConversionResponseFile() (string, error) {
 
 	return conversionPath, nil
 }
+
+func CreateRateLimiter(cfg *config.HookConfig) *rate.Limiter {
```

Comment:
What do you think about returning nil from this method and then calling `Wait` only if RateLimitter is not `nil`? It will help us to avoid unnecessary interactions.

Reply:
I thought about it. The default RateLimiter has "Inf" limit, so `Wait` returns immediately. We can hide limiter inside RateLimitWait (https://github.com/flant/addon-operator/pull/176/files#r586508346) and do `nil` checking there.